.github/actions/prefix-pr/Prefix-PR.ps1

```diff
@@ -0,0 +1,46 @@
+param(
+    [string] $Github_Repository,
+    [string] $Github_Ref,
+    [string] $Branch,
+    [string] $Github_Token
+)
+
+$jiraBaseUrl = "https://lombiq.atlassian.net/browse/"
+$owner, $repo = $Github_Repository.Split('/')
+$prId = $Github_Ref -replace "refs\/pull\/(\d+)\/merge", '$1'
+
+$url = "https://api.github.com/repos/$owner/$repo/pulls/$prId"
+$headers = @{"Authorization" = "Bearer $Github_Token"; "Accept" = "application/vnd.github+json"}
+
+$pr = Invoke-WebRequest $url -Headers $headers | % {$_.Content} | ConvertFrom-Json
+$title = $pr.title
+$body = $pr.body
```

Comment:
These are available in the [`github.event.pull_request` context](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request) already. `$prId` is contained there too.